utils/check_cross_ref_images.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
I executed that script on every md files and it reports bizzare things.  For example, in 2018-11-08-CNN-VAE.md, every image are correct except for sc04.png and sc05.png which do exit.  After looking into the md file, I found a space at the end of the sc04.png file and a dot at the end of the sc05.png files.  The space and the dot we added to the file names which then could'nt be found

Reply:
Thanks for the review ! Will try to have a look at that.

Reply:
There were a few issues here:
- It looks like we had left behind a handful of cross-references to some files in #359. PR to fix them: #378
- As for the rest, the script did not handle two corner cases: commit f74dab0 addresses them.